packages/block-editor/src/components/link-control/constants.js

```diff
@@ -24,4 +24,10 @@ export const DEFAULT_LINK_SETTINGS = [
 		id: 'opensInNewTab',
 		title: __( 'Open in new tab' ),
 	},
+	{
+		id: 'markAsNoFollow',
+		title: __(
+			'Search engines should ignore this link (mark as nofollow)'
+		),
```

Comment:
```suggestion
		title: __(
			'No follow'
		),
		help: __(
			'Search engines should ignore this link'
		),
```

Let's extend this and then make changes within `settings.js` to _optionally_ include the `help` prop on the `ToggleControl`.